frontend/packages/shared/src/navigation/PackagesRouter/PackagesRouter.test.ts

```diff
@@ -0,0 +1,89 @@
+import { PackagesRouter } from './PackagesRouter';
+
+const mockOrg: string = 'org';
+const mockApp: string = 'app';
+
+describe('PackagesRouter', () => {
+  describe('constructor', () => {
+    it('should initialize app and org properties', () => {
+      const packagesRouter = new PackagesRouter({ org: mockOrg, app: mockApp });
+
+      expect(packagesRouter['app']).toEqual(mockApp);
+      expect(packagesRouter['org']).toEqual(mockOrg);
+    });
+
+    it('should default to empty strings if app and org are not provided', () => {
+      const routerWithoutParams = new PackagesRouter({});
+      expect(routerWithoutParams['app']).toEqual('');
+      expect(routerWithoutParams['org']).toEqual('');
```

Comment:
How about assessing this by examining the exposed public methods? 🤔 If these methods return the expected values/outcome, it indicates that our implementation has undergone thorough testing.

```suggestion
      const routerWithoutParams = new PackagesRouter();
      const route = routerWithoutParams.getNavigationUrl('somethig');
      expect(route).toBe('something-without-params');
```

Reply:
I agree, but then we probably need to change the params in the constructor to be optional as well? 
I'm getting this error when doing what you suggest: 
<img width="533" alt="image" src="https://github.com/Altinn/altinn-studio/assets/133344438/d359f011-5e4d-4f5c-81b9-cf926d571aaa">

Reply:
Sure, the params should be optional. 😊

Reply:
Ref. the [this comment ](https://github.com/Altinn/altinn-studio/pull/11883/files#r1431112603),  we can now test it as I suggested initially in this comment. :)